api/organization/donation-delete.js

```diff
@@ -1,4 +1,4 @@
-const { ORG_ROLES, MSGS: { INTERNAL_SERVER_ERROR, NO_DONATION, INSUFFICIENT_PERMISSIONS } } = require('../../helpers/constants')
+const { MSGS: { INTERNAL_SERVER_ERROR, NO_DONATION, INSUFFICIENT_PERMISSIONS } } = require('../../helpers/constants') // eslint-disable-line
```

Comment:
saddddddddddddd; nit: shouldn't have eslint disabled here. does not block merge

Reply:
idk why eslint_disables are added. i don't do this manually, maybe when i run npx standard it's adding them?